pages/ksphere/kommander/1.1.0-beta/operations/infrastructure-providers/configure-aws-infrastructure-provider-roles/index.md

```diff
@@ -175,7 +175,9 @@ To use the role created attach the following policy to the role which is already
 }
 ```
 
-Instead of doing this manually you can add this file, `iam-stsAssumeKommander.tf`, into the `extras/provisioner/` directory next to your `cluster.yaml` file.
+#### Create role when creating the management cluster
+
+Instead of doing this manually you can create the following file, name it, `iam-stsAssumeKommander.tf`, into the `extras/provisioner/` directory next to your `cluster.yaml` file. This will create the role when you create or update your cluster with `konvoy up`. Ensure you replace `THEROLEYOUCREATED` with the AWS Role name.
```

Comment:
```suggestion
Instead of doing this manually create the following file, `iam-stsAssumeKommander.tf`, int the `extras/provisioner/` directory, next to your `cluster.yaml` file. This creates the role when you create or update your cluster with `konvoy up`. Ensure you replace `THEROLEYOUCREATED` with the AWS Role name.
```

Reply:
`int` ? do you mean `in`